lib/vow.js

```diff
@@ -60,7 +60,7 @@ var undef,
             }
 
             if(isPostMessageAsync) {
-                var msg = '__promise' + +new Date,
+                var msg = '__promise' + Math.random() + '_' +new Date,
```

Comment:
I was just passing by and noticed that a plus sign is lost before the `+new Date`. Probably it was intended to be `'__promise' + Math.random() + '_' + +new Date`